src/main/java/org/jilt/internal/AbstractBuilderGenerator.java

```diff
@@ -331,12 +331,18 @@ protected final TypeVariableName mangleTypeParameter(TypeVariableName typeVariab
     }
 
     private ParameterSpec setterParameterSpec(VariableElement attribute, TypeName parameterType) {
-        ParameterSpec.Builder ret = ParameterSpec.builder(parameterType,
-                this.attributeSimpleName(attribute));
-        AnnotationMirror nullableAnnotation = this.firstAnnotationCalledNullable(attribute);
+        ParameterSpec.Builder ret = ParameterSpec.builder(parameterType, this.attributeSimpleName(attribute));
+        AnnotationMirror nullableAnnotation = firstAnnotationCalledNullable(attribute);
```

Comment:
Can you please revert these changes? They are functionally the same, so they're just making the diff in the PR bigger.

```suggestion
        ParameterSpec.Builder ret = ParameterSpec.builder(parameterType,
                this.attributeSimpleName(attribute));
        AnnotationMirror nullableAnnotation = this.firstAnnotationCalledNullable(attribute);
```

Reply:
Sorry about this!

Reply:
No worries 🙂. Thanks for the revert!